src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -257,8 +257,8 @@ protected static bool IsFieldOrPropertyReference(IOperation operation, INamedTyp
 
         protected class MyCodeAction : CodeAction.DocumentChangeAction
         {
-            public MyCodeAction(string title, Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(title, createChangedDocument, equivalenceKey: title)
+            public MyCodeAction(string title, Func<CancellationToken, Task<Document>> createChangedDocument, string? equivalenceKey = null)
```

Comment:
Why is this optional parameter? Are there any callers that don't pass an `equivalenceKey`?

Reply:
Yes, there is another class that inherits from this, and I didn't want to update them without an editor that can validate I'm doing it right. This is lazy Friday night changes :)